pkg/pods/pod.go

```diff
@@ -193,6 +202,21 @@ func (pod *Pod) Launch(manifest Manifest) (bool, error) {
 	return success, nil
 }
 
+func (pod *Pod) Prune(max size.ByteCount, manifest Manifest) error {
+	launchables, err := pod.Launchables(manifest)
+	if err != nil {
+		return err
+	}
+	for _, l := range launchables {
+		err = l.Prune(max)
+		if err != nil {
+			pod.logLaunchableError(l.ID(), err, "Could not prune directory")
+			// Don't return here. We want to prune other launchables if possible.
+		}
+	}
+	return err
```

Comment:
if I have three launchables and they returned in order no error, error, no error, this function would return no error, is that OK? I have a feeling it should return the error sent by the second prune.

(It will be logged correctly of course, so that's fine, but should probably return the error too)